<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BalaCheck</class>
 <widget class="QWidget" name="BalaCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>502</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>My Finance</string>
  </property>
  <widget class="QTabWidget" name="pageSelector">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="monthsPage">
    <attribute name="title">
     <string>Month Payments</string>
    </attribute>
    <widget class="QTableWidget" name="currentMonth">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>691</width>
       <height>401</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>2</number>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>Bill from</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Due date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Payed on</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>To pay</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Payed by</string>
      </property>
     </column>
    </widget>
    <widget class="QTabWidget" name="monthsList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>691</width>
       <height>21</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="monthDec2019">
      <attribute name="title">
       <string>2019, Dec</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="banksPage">
    <attribute name="title">
     <string>Banks</string>
    </attribute>
    <widget class="QTableWidget" name="transactionsLog">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>541</width>
       <height>241</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Info</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="scaduledTransLog">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>300</y>
       <width>541</width>
       <height>161</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Scheduled</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Info</string>
      </property>
     </column>
    </widget>
    <widget class="QTabWidget" name="banksList">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>481</width>
       <height>21</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>{No banks}</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Transactions</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="transTotal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sceduled</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Transactions</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bottomTotal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>661</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="addBank">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add bank</string>
     </property>
    </widget>
    <widget class="QToolButton" name="deleteBank">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete bank</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="paymentsPage">
    <attribute name="title">
     <string>Payments</string>
    </attribute>
    <widget class="QTableWidget" name="paymentsList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>521</width>
       <height>411</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Paying to</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Due date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Period</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>AutoPay from</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="addPaymentBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add payment</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resorces/add-icon.png</normaloff>../resorces/add-icon.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="deletePaymentBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove payment</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resorces/delete-icon.png</normaloff>../resorces/delete-icon.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="incomesPage">
    <attribute name="title">
     <string>Incomes</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>651</width>
       <height>141</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Payed by</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Pay day</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Priod</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Method</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>651</width>
       <height>241</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Payed by</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Pay day</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Method</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Recurrent payments</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>One time payments</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add payment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>10</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove payment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>190</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add payment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>190</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove payment</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
